lib/drivers/roche/accuChekUSB.js

```diff
@@ -0,0 +1,592 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2019, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import _ from 'lodash';
+import sundial from 'sundial';
+
+import annotate from '../../eventAnnotations';
+import common from '../../commonFunctions';
+import TZOUtil from '../../TimezoneOffsetUtil';
+import UsbDevice from '../../usbDevice';
+import {
+  MDC_PART_OBJ,
+  DATA_RESPONSE,
+  ACTION_TYPE,
+  APDU_TYPE,
+  DATA_ADPU,
+  EVENT_TYPE,
+  getObject,
+  getAttribute,
+  getAttributeList,
+  getProductionSpecEntry,
+} from './utils';
+
+const isBrowser = typeof window !== 'undefined';
+// eslint-disable-next-line no-console
+const debug = isBrowser ? require('bows')('AccuChekUSBDriver') : console.log;
+
+class AccuChekUSB {
+  constructor(cfg) {
+    this.cfg = cfg;
+  }
+
+  static get TIMEOUT() {
+    return 5000;
+  }
+
+  async openDevice(deviceInfo, cb) {
+    this.usbDevice = new UsbDevice(deviceInfo);
+    this.usbDevice.device.open(false); // don't auto-configure
+    this.usbDevice.device.reset();
+
+    this.usbDevice.device.setConfiguration(1, async () => {
+      if (this.usbDevice.device.interfaces == null) {
+        throw new Error('Please unplug device and retry.');
+      }
+
+      [this.usbDevice.iface] = this.usbDevice.device.interfaces;
+      this.usbDevice.iface.claim();
+
+      /* eslint-disable no-bitwise */
```

Comment:
Should this be a `eslint-disable-next-line no-bitwise` on the following line?

Reply:
Hmm, that's what I should've done, yes. Although I would also support changing `no-bitwise` to just a warning in our eslint config files, because we need to do bitwise operations in driver code quite frequently. I will fix it in another branch, as I don't want to go through another review-test cycle for this :wink: